.eslintrc.json

```diff
@@ -169,6 +169,17 @@
           "off"
         ]
       }
+    },
+    {
+      "files": [
+        "webpack/*",
+        "**/__factories__/*",
+        "*.stories.js",
+        "*.test.js"
+      ],
+      "rules": {
+        "sitekit/no-direct-date": "off"
```

Comment:
Do we actually want to disable this in all these places? We've seen tests become unstable in the past due to timezone influence. I realize this is what we had before which has moved, I just think it's worth double-checking that we should be disabling the rule in each of these.

Reply:
Fair point, I'll take a look 🤔

Reply:
Thanks for checking